pkg/endpoint/endpoint.go

```diff
@@ -1675,6 +1678,10 @@ func (e *Endpoint) replaceInformationLabels(l pkgLabels.Labels) {
 // returned.
 // Must be called with e.Mutex.Lock().
 func (e *Endpoint) replaceIdentityLabels(l pkgLabels.Labels) int {
+	if l == nil {
```

Comment:
I was worried that this changes behavior as nil no longer removes all labels but I have not found a caller that previously passed in nil.

Please add a comment to the function that indicates that if l is nil, no update is performed.